scarlet/display.py

```diff
@@ -146,12 +141,10 @@ def img_to_3channel(img, channel_map=None, fill_value=0):
     return rgb
 
 
-def img_to_rgb(img, channel_map=None, fill_value=0, norm=None):
+def img_to_rgb(img, channel_map=None, fill_value=0, norm=None, mask=None):
```

Comment:
why is `mask` given to this function? it can work happily without as it doesn't affect the conversion from pixels to RGB at all

Reply:
It's a convenience to mask pixels in the function instead of having to do it manually, otherwise we need to add the `dstack` method to `show_scene` and users would have to manually mask the image.

More importantly it will always mask the image when a pixel is masked in every band, which can be useful to know if one does not know that pixels were masked.